tests/test_hierarchy_helpers.hpp

```diff
@@ -34,6 +34,12 @@
 #include "laplace.hpp"
 #include "laplace_matrix_free.hpp"
 
+// If activated the global operator's ShapeInfo object is copied instead of
+// recalculated when initializing the agglomerate operators. Requires that
+// deal.II is patched with scripts/0002-Avoid-copying-shape-info.patch.
+//
+// #define USE_OPTIMIZED_AGGLOMERATE_INITIALIZATION
```

Comment:
How did you intent to enable the optimization?
One option would have been to define it in deal.II's header you patched but it involves rebuilding the image.
Another might be adding the compile definition using CMake.

Reply:
Just uncomment and recompile. I don't expect us to switch between the two often, i.e., performance test,  you uncomment otherwise you don't care